.github/workflows/setup.sh

```diff
@@ -0,0 +1,23 @@
+MS_SUITE="omero-ms-zarr-suite"
+BF_SUITE="ome-zarr-bf2raw-suite"
+ZARR_SUITE="ome-zarr-py-suite"
+
+REPO=$1
+# Checkout the sha from the specified repository.
```

Comment:
This is done by configure_test.py? Or can this comment be dropped?

Reply:
Actually this is no longer needed since we fetch head.
Thanks